distribution/docs/src/main/resources/_contents/_solr-contents/managing-solr-contents.adoc

```diff
@@ -309,3 +309,53 @@ Here are some helpful suggestions for setting up data backups and recoveries:
 * Run a scheduler program that calls the backup URL on a timed basis.
 * Put indicators in place that can detect when data corruption may have occurred.
 * Testing a backup before recovery is possible. A replicated "staging" Solr server instance can be stood up, and the backup can be copied to that system for testing before moving it to the "production" system.
+
+=== High Availability (Optional)
+
+Availability is used to describe the period of time when a service is available, as well as the time required by a system to respond to a request made by a user.
+High availability refers to systems that are durable and likely to operate continuously without failure for a long time.
+
+==== SolrCloud
+
+SolrCloud provides distributed capabilities needed by SOLR servers to communicate and establish a highly available, fault tolerant cluster of SOLR instances.
+
+===== Prerequisites for SolrCloud
+
+Solr 6
+
+CentOS 6 or greater
```

Comment:
@oconnormi, @figliold, should we put CentOS as a pre-requisite or just have a note saying that the instructions have been tested on CentOS? Technically, the instructions are fairly generic and should work on almost any UNIX-based system, right?